### DISCUSSION TITLE
I copied the Github Repository and I keep getting this error when I try and run the bot. I'm confused why.

### DISCUSSION THREAD

**Discussion by UMMMMMMMMMM2945** [DoobDev/Doob#176] [Need Help] [answered]
https://pastebin.com/5aub2H8y

The only things I have changed is the stuff in .env_example to have the API keys like it said in the comments.

**Comment by mmattbtw** [comment, marked as answer]
Have you changed it from `.env_example` to `.env`?

Also move `servercount.py`, `stat.py`, and `music.py` to the `NOT IN USE` folder.

**Reply by UMMMMMMMMMM2945** [reply]
oh that was my issue oops. Thank you.
